README.md

```diff
@@ -1,73 +1,91 @@
-# Pathable vNext Code Challenge
+# Pathable Code Challenge
+
 Do you want to join Pathable as a developer? Great. Read this document and then submit your solution. Good luck.
 
-# Machine setup
+## Machine setup
+
 - [Install Meteor](https://www.meteor.com/install)
 
-# Challenge setup
-## Repository setup
-The challenge setup is part of the challenge then we are not going to help or answer questions. Follow the steps exactly as they are described.
+## Challenge setup
+
+### Repository setup
+
+Please following the instructions here exactly. Getting setup correctly is part of the challenge! ;)
+
 - This git repository needs to be used as your initial state.
-- Publish your version of this repository in somewhere that is private
-  - It needs to be private because we don't want your solution to be public
-    - You can publish it where you prefer. Gitlab and Bitbucket are free for private repository with a single developer
-  - Allow these public keys ssh keys to access your repository
+- Please publish your version of this repository somewhere private
+  - IMPORTANT: Your solution should not be public.
+    - You can publish it anywhere you prefer. Gitlab and Bitbucket are free for private repository with a single developer
+  - Please add these public SSH keys:
+
     ```bash
         ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCqQZDC4RV1Zyw7f1XFYAiaEYFXD8j2cHi5Sg/eVyq8ts3WIdEqRET0Vgdu52cAQdhq12FBMOEklqki2szid+8woIBcevwz+6wo19WNt7JHXWFuic+oR39Bplw41elKoAx7APo135nP/z1BolcywXBFUlA72PPytrpbgeic6ZYRk2Df+DRq+o0Jl0wexs7E9F0AK6B4oRQ8Ybq6I8odT7Hl5jIW0sXQzzGvdgOsaV4W+SqzPFai2y1FZs5A0bA/3JRhielLR8Hyin2lp6FTom7TrSbWCbRj+PUX4mGs3tHYydl+m4rFC4F/CPtxnrcbGog6wtCgmNCifXe9d/TY/AtH jenkins@ip-172-31-91-146 
     ```
+
     ```bash
         ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDQFVf3TXBOoUyw3GXCRCkM85JYnSx+MT4W7d3bm7RZ7p9xZG9QcQRSvVDg3uc0Fjt2ujzxditMW6epcn5irtByuW55TnKSbCUl9Xl0jlqgCW3GDtP1BCE6xH2iuUjQSSHCFSD7CqjgK4zAdgpOY3r3oZtMoOmHrQDyOGBG4XAQz6LE9u+E3NBLf5STBpR+uvuFAzIimcGVhyeznbyPtUELgHDyNFIlMqzAr7chl8m/x/zm808FLnE3oIUgg+80IbgbF3fmktiNdNgCE6gWwZniVe3avDUX1o2g8BqtYuyFgETugQBy31Y/mZoPfzl2DE7nR8+QF/l4IAW4Owt4XJS5 filipe-macbook 
     ```
+
 - Remember to push everything that you have locally before submitting your challenge. We will only use content that is available in your published repository.
 
-## How to install dependencies
+### How to install dependencies
+
 ```bash
 meteor npm install -g yarn
-meteor yarn install 
+meteor yarn install
 ```
- 
-## How to run
+
+### How to run
+
 ```bash
 meteor yarn start
 ```
 
-## How to run cypress tests
+### How to run tests
```

Comment:
👍 

I was thinking about also setting up Jest but in the end, I decided to let the candidates set up if they want.